addons/event_sale/models/event.py

```diff
@@ -21,6 +21,7 @@ class event_event(models.Model):
     badge_back = fields.Html('Badge Back', translate=True, states={'done': [('readonly', True)]})
     badge_innerleft = fields.Html('Badge Innner Left', translate=True, states={'done': [('readonly', True)]})
     badge_innerright = fields.Html('Badge Inner Right', translate=True, states={'done': [('readonly', True)]})
+    is_unlimited = fields.Boolean('Unlimited Tickets', compute='_compute_seats_availability')
```

Comment:
Don't understand. We already have a seats_availability field. If I understand well, the purpose is to delegate the limited / unlimited computation to tickets. I would advise you to override the seats_availability field of the event, adding a _compute and store so that this field value is not computed based on tickets, instead of adding a new field that will conflict with the old one. What do you think about that ?